build-configs/themes/integreat/fonts.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+
+import type { FontsType } from '../common/fonts'
+import { commonFonts } from '../common/fonts'
+
+const fonts: FontsType = {
+  decorativeFontBold: 'Raleway-Bold',
+  decorativeFontRegular: 'Raleway-Regular',
+  contentFontRegular: 'OpenSans-Regular',
+  contentFontBold: 'OpenSans-Bold',
+  arabicWebviewFontFamilies: '\'OpenSans\', \'Lateef\', sans-serif',
+  webviewFontFamilies: '\'OpenSans\', sans-serif',
```

Comment:
Afaics, these are also common, aren't they?

Reply:
Right now they are, but this will probably be changed with malte.